specifyweb/frontend/js_src/lib/hooks/useValidation.tsx

```diff
@@ -115,7 +115,7 @@ export function useValidation<T extends Input = Input>(
     validationRef: React.useCallback(
       (input): void => {
         inputRef.current = input;
-        setValidation(validationMessageRef.current, 'focus');
+        setValidation(validationMessageRef.current, 'auto');
```

Comment:
while this change here may fix the issue, it also changes the behavior in a lot of places (useValidation is used in a lot of places)

thus, you need to test the impact of this change in all the places where validationRef is used

it is possible that this change in behavior is desirable in all cases - but we need to check it, not just assume it is so

Reply:
tested.